src/config/thorough.ts

```diff
@@ -26,7 +26,7 @@ export const overridesWhenThorough = (function () {
       },
       rules: {
         "@typescript-eslint/await-thenable": "error",
-        "@typescript-eslint/no-floating-promises": "error",
+        "no-floating-promises-qawolf": "error",
```

Comment:
```suggestion
        "@qawolf/no-floating-promises-qawolf": "error",
```

Not sure whether we need "@qawolf/" here. Haven't thought of a way to test this PR yet.

Reply:
To test npm packages before releasing them, here’s a universal recipe:

1. Build your package you’re editing (e.g. `npm run build`).
2. Package the built version with `npm pack`.
3. Move into the app you want to use this version in (e.g. `cd mydev/qawolf/platform`).
4. Install the packaged version with `npm install mydev/the-other-package/the-other-package.1.2.3.tgz`

Reply:
Proof of it working:

![CleanShot 2025-02-13 at 14 23 30@2x](https://github.com/user-attachments/assets/33ad76cd-43a0-43a4-b85a-2245f1de0aaa)

![CleanShot 2025-02-13 at 14 26 14@2x](https://github.com/user-attachments/assets/2f27c12a-69b7-413c-943e-806f3e3e6fa4)